### DISCUSSION TITLE
Github Paketleri

### DISCUSSION THREAD

**Discussion by erenndev** [community/community#19504] [Actions] [answered]
Merhaba, sorum github paketlerine arkadaşlarımı nasıl yetkilendirebilirim ve görünürlüğünü nasıl public yapabilirim ?

**Comment by Huseyin-Cinar** [comment, marked as answer]
Merhaba @erenndev,

- Github paketlerine arkadaşlarını eklemek için paketini seç, paket ayarlarına tıkla, ekipleri veya kişileri davet et tıkla ve rol menüsünden yetkisini seç.
- Github paketlerinin görünürlüğünü değiştirmek için paketini seç, paket ayarlarına tıkla, en altta tehlikeli bölgeye gel ve make public yap.

- Daha fazlası için https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility#configuring-access-to-container-images-for-your-personal-account okuyabilirsin.

**Reply by Huseyin-Cinar** [reply]
Yanıt olarak işaretlemeyi unutma :)
